test/exchange/UniswapExchange.02.test.ts

```diff
@@ -0,0 +1,208 @@
+import chai = require('chai');
+// tslint:disable-next-line:no-var-requires
+const { balance, BN, constants, ether, expectEvent, expectRevert, send } = require('@openzeppelin/test-helpers');
+import {
+    TestERC20MintableInstance,
+    TestUniswapExchangeInstance,
+    TestUniswapFactoryInstance,
+} from '../../types/truffle-contracts';
+
+// tslint:disable:max-line-length
```

Comment:
Avoid disabling this rule. I do have a big screen and often don't really mind, but sometimes I do work just on the laptop and some other people do as well. It's a really bad experience to be scrolling around.